OracleLinux/7/.env

```diff
@@ -0,0 +1,16 @@
+# Oracle Linux 7 configuration file
+#
+# Requires vagrant-env plugin
+#
+# This file will be overwritten on updates, it is recommended to make changes
+# in .env.local
+
+# Extensions
+# To extend the box, uncomment one of the following:
+
+# LAMP server
+# EXTEND=lamp
+# EXPOSE=8080:80
+
+# Docker engine
+# EXTEND=docker-engine
```

Comment:
Should this not be `EXTEND=$EXTEND,docker-engine` to allow for multiple extensions? I would also rename it to `container-runtime` (see below for the more detailed reason).